ui/vite.config.ts

```diff
@@ -107,6 +107,45 @@ export default ({ mode }: { mode: string }) => {
   console.log(process.env.APP);
   console.log(mode, app, base(mode, app));
 
+  const rollupOptions = {
+    external:
+      mode === 'mock' || mode === 'staging'
+        ? ['virtual:pwa-register/react']
```

Comment:
Why do we need this in mock/staging?

Reply:
was running into type errors from the SW since this is in the update banner